README.md

```diff
@@ -21,83 +21,83 @@ Sensitive Tree.
 
 To achieve the first goal, the primary output of this repository is a
 static website which allows the reader to understand the definitions
-contained here, and link to other online resources where more information
+contained there and link to other online resources where more information
 can be obtained.
 
 Links to Wikidata, Antlr definitions, E(BNF) files, example files, will
-be integral components of the definitions here.
+be integral components of the definitions there.
 
 Terminology used to describe language components will be consistent
-across languages where-ever possible, and defer to terminology used
-in academic literature or study guides, to make these definitions more
+across languages wherever possible, and defer to terminology used
+in academic literature or study guides to make these definitions more
 accessible and useful to students of language theory.
 
 ## Stages
 
-1. Organically grow a human readable fact based database of any syntax,
-   stored in YAML files, covering any language from large and complicated
-   programming languages down to strings like a URL, especially focusing
-   on style description which describe a subset of a language.
+1. Organically grow a human-readable fact-based database of any syntax --
+   stored in YAML files -- covering any language, from large and complicated
+   programming languages down to strings like URLs, especially focusing
+   on style descriptions which describe a subset of a language.
 
 2. Create programs to load these definitions and convert input files
-   into a universal AST, primarily for building a test suite to verify the
-   language definitions are able to parse files at useful levels of detail,
-   again focusing on style-defined subsets of languages which are easier
-   and also more useful.
+   into a universal AST, primarily for building a test suite to verify that
+   the language definitions are able to parse files at useful levels of
+   detail. Again, focusing on style-defined subsets of languages, which are
```

Comment:
I can see why you added a period and capitalized `Again`, but there is now a new sentence fragment that should be clarified. maybe change `focusing` to `focus`, so that the fragment becomes a complete sentence.